assets/src/dashboard/app/api/useFontApi.js

```diff
@@ -27,7 +27,7 @@ const useFontApi = (dataAdapter, { wpApi }) => {
 
     return dataAdapter.get(wpApi).then((data) =>
       data.map((font) => ({
-        value: font.name,
+        value: font.family,
```

Comment:
Is this broken already now? If so, it looks like something we need to submit ASAP?

Reply:
Not broken AFAIK, just inconsistent